### DISCUSSION TITLE
升级开发镜像的版本

### DISCUSSION THREAD

**Discussion by cyenxchen** [alibaba/ilogtail#988] [Ideas]
vscode有个插件叫`CodeLLDB`，利用该插件可以使用vscode的图形界面调试工具，能大大方便ilogtail的核心代码调试，但是该插件依赖glibc-2.18以上的版本，官方提供的开发镜像是centos7，glibc版本太低导致该插件用不了。虽然有临时[解决办法](https://gist.github.com/JaySon-Huang/63dcc6c011feb5bd6deb1ef0cf1a9b96)，但毕竟不方便。希望官方可以考虑升级下开发镜像发行版版本。

**Comment by yyuuttaaoo** [comment]
这个工作我们预计下个月会着手推进，我们也想把镜像和gcc版本都升级一下，可以关注一下。这个插件可以推荐到 #299 中吗？

**Reply by cyenxchen** [reply]
ok

**Reply by messixukejia** [reply]
升级后，欢迎来双周会分享下。

**Reply by cyenxchen** [reply]
分享就不必了吧，没多少内容，到时候我可以完善下你们的开发文档。
